packages/create-app/src/index.ts

```diff
@@ -39,8 +39,19 @@ const main = (argv: string[]) => {
       '--skip-install',
       'Skip the install and builds steps after creating the app',
     )
+    .addOption(
+      new Option(
+        '--template-path [directory]',
+        'Use an external application template instead of the default template',
+      ).conflicts('--template-name'),
+    )
+    .addOption(
+      new Option('--template-name [name]', 'Use the given template')
```

Comment:
I have a prepared commit to add a second one: `default-next-app` using the experimental `backend-next` instead of the legacy one.
Do you think I could add this commit to this PR or to a followup PR ?

Reply:
In fact I removed the `--template-name` option from this PR, and added it to the followup PR #18645 which will also add the alternate `default-next-app` application template